library/src/main/java/de/agilecoders/wicket/webjars/request/resource/WebjarsCssResourceReference.java

```diff
@@ -25,7 +25,7 @@
      * @param name The webjars path to load
      */
     public WebjarsCssResourceReference(final String name) {
-        super(WebjarsCssResourceReference.class, useRecent(name));
+        super(WebjarsCssResourceReference.class, prependWebjarsPathIfMissing(name));
```

Comment:
This is the reason for the broken test in wicket-bootstrap project.
With this change the rendered urls use `.../current/...` instead of `.../4.1.2/...`

Reply:
Fixed with https://github.com/l0rdn1kk0n/wicket-webjars/commit/dc437eea069d4bd3c226b50cfdc942d8702b3441